storybookjs/storybook
# SWITCH to circleci over travisCI && CHANGE lerna bootstrap procedure:

Issue:
- TravisCI is taking a long time to build 
- Fixes: #1127
- Bootstrapping takes a long time
- A lot of modules are duplicated during bootstrap
  Because of this modules are not singletons during development.
  This leads to a list of problems, like duplicate react, no channel in `lib/addons`..
- Storyshots testing is disabled in cra-kitchen-sink

## What I did
- Configure CircleCI (in parallel)
- Remove TravisCI
- Use yarn for installation of packages
- Hoist all dependencies to root using `--hoist`
- Hoist known packages to root using a custom script
- Test storyshots in cra-kitchen-sink

## How to test
- This branch should be build with CircleCI
- Doing a clean install and bootstrap on this branch should be successful
- After bootstrap unit tests should pass
- After bootstrap starting `example/cra-kitchen-sink` should be successful
- You should feel a feeling of relief and inner happiness knowing how much faster the CI is.
